gamestonk_terminal/portfolio/port_controller.py

```diff
@@ -0,0 +1,161 @@
+__docformat__ = "numpy"
+
+import argparse
+from typing import List
+from prompt_toolkit.completion import NestedCompleter
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.helper_funcs import get_flair
+from gamestonk_terminal.menu import session
+from gamestonk_terminal.portfolio import rh_api
+
+
+class PortfolioController:
+
+    CHOICES = [
+        "help",
+        "q",
+        "quit",
+        "login",
+        "hold",
+        "rhhist",
+    ]
+
+    BROKERS = [
+        "rh",
+    ]
+
+    def __init__(self):
+        self.port_parser = argparse.ArgumentParser(add_help=False, prog="port")
+        self.port_parser.add_argument("cmd", choices=self.CHOICES)
+        self.brokers_list = set()
+
+    @staticmethod
+    def print_help(broker_list):
+        if not broker_list:
```

Comment:
You don't need these 7 lines, look at my next comment

Reply:
I assume you mean starting at `if not broker_list` ?  It should be a static method, right?

Reply:
Yea, I mean that the line of code that I mentioned below can do this all in 1 go :p